misc/scripts/install_d3d12_sdk_windows.py

```diff
@@ -70,6 +71,16 @@
 print(f"Mesa NIR {mesa_filename} installed successfully.\n")
 
 # WinPixEventRuntime
+
+# MinGW needs DLLs converted with dlltool.
+# We rely on finding gendef/dlltool to detect if we have MinGW.
+# Check existence of needed tools for generating mingw library.
+gendef = shutil.which("gendef") or ""
+dlltool = shutil.which("dlltool") or ""
+if dlltool == "":
+    dlltool = shutil.which("x86_64-w64-mingw32-dlltool") or ""
+has_mingw = gendef != "" and dlltool != ""
```

Comment:
Should there be a way to opt out of MinGW support, or keep both MinGW and MSVC-friendly copies of WinPixEventRuntime alongside? If you have MinGW installed but typically use MSVC, this means you won't be able to use MSVC to build Godot with D3D12 enabled.

Reply:
Does it? AFAIK this just generates additional files, the files used by MSVC should still be there.

Reply:
MinGW library is `.a` and MSVC is `.lib`, it should not conflict.